server/alerts_email_via_python3.py

```diff
@@ -0,0 +1,273 @@
+</span></p></td></tr></table></td></tr><tr><td style='padding:.75pt .75pt .75pt .75pt'>   <tr>
+<td width="100%" style='width:100.0%;border:none;padding:7.5pt .15in 7.5pt .15in;min-width: 315px;max-width: 660px'>
+<div align=center><table class=MsoNormalTable border=0 cellspacing=3 cellpadding=0 width="100%" style='width:100.0%'><tr>
+<td style='padding:.75pt .75pt .75pt .75pt;color:white'><p class=MsoNormal>Powered by: 
+<a href="https://github.com/securestate/king-phisher">RSM US, LLP</a> 
+<o:p></o:p></p></td></tr></table></div></td></tr></table><div class="Footer">
+<img src="https://github.com/securestate/king-phisher/raw/master/data/king-phisher-logo.png">
+</div>
+<p class=MsoNormal><o:p>&nbsp;</o:p></p></div></body></html>
+"""
+
+class Plugin(plugins.ServerPlugin):
+	authors = ['Austin DeFrancesco', 'Spencer McIntyre', 'Mike Stringer', 'Erik Daguerre']
+	classifiers = ['Plugin :: Server :: Notifications :: Alerts']
+	title = 'Campaign Alerts: via Python 3 SMTPLib'
+	description = """
+	Send campaign alerts via the SMTP Python 3 lib. This requires that users specify
+	their email through the King Phisher client to subscribe to notifications.
+	"""
+	homepage = 'https://github.com/securestate/king-phisher-plugins'
+
+	# Email accounts with 2FA, such as Gmail, will not work unless "less secure apps" are allowed
+	# Reference: https://support.google.com/accounts/answer/60610255
+	# Gmail and other providers require SSL on port 465, TLS will start with the activation of SSL
+	options = [
+		plugin_opts.OptionString(
+			name='smtp_host',
+			description='Location of SMTP server',
+			default='localhost'
+		),
+		plugin_opts.OptionInteger(
+			name='smtp_port',
+			description='Port used for SMTP server',
+			default=25
+		),
+		plugin_opts.OptionString(
+			name='smtp_email',
+			description='SMTP email address to send notifications from',
+			default=''
+		),
+		plugin_opts.OptionString(
+			name='smtp_password',
+			description='Password associated with smtp email',
+			default=''
+		),
+		plugin_opts.OptionBoolean(
+			name='ssl_tls',
+			description='Set connection to smtp server to use ssl/tls connection',
+			default=True
+		),
+		plugin_opts.OptionString(
+			name='email_jinja_template',
+			description='Password associated with smtp email',
+			default=''
+		),
+		plugin_opts.OptionString(
+			name='mail_options',
+			description='Additional mail options',
+			default=''
+		),
+	]
+	req_min_version = '1.3'
+	req_min_py_version = '3.0'
+	req_packages = {
+		'smtplib': has_smtp
+	}
+
+	def initialize(self):
+		signals.campaign_alert.connect(self.on_campaign_alert)
+		return True
+
+	def on_campaign_alert(self, table, alert_subscription, count):
+		user = alert_subscription.user
+		if not self.config['smtp_email']:
+			self.logger.debug("user {0} has no email address specified, skipping SMTP alert".format(user.id))
+			return False
+		if self.send_message(table, alert_subscription, count, self.config['smtp_email']):
```

Comment:
Since `send_message` returns a boolean you could simply do `return self.send_message(table,...` here and cut the logic that's not really doing anything.